### PR TITLE
Modified password validation

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
Improved user experience for password validation #247

### CODE REVIEW

File: systers_portal/users/adapter.py [hunk 1]
```diff
@@ -23,34 +23,9 @@ def clean_password(self, password):
...
-
-            elif not digit and special_char and uppercase:
-                raise ValidationError("Password must have atleast one digit")
-
             else:
                 raise ValidationError(
-                    "Password must have atleast one uppercase letter," +
-                    " one special character and one digit")
-
+                    "Password must have at least 6 characters and must have at least one uppercase letter, one special character and one digit.")
```
Comment (tapasweni-pathak) [inline systers_portal/users/adapter.py]: Please change this to *Password must have at least, 6 characters, one uppercase letter, one special character and one digit.*

**Comment by tapasweni-pathak** [comment]
Please squash the commit messages [Screenshot from Gyazo](https://gyazo.com/8971e514b7e065399d4d7910b4420fc4) into one. Say *Show all necessary check for password strength*. or if you can come up with a better one, please go ahead.

**Comment by Himanshi-Khandelwal** [comment]
Done :)

**Comment by tapasweni-pathak** [comment]
Cool, always flake8 your changes.

**Pull request merged**